### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by JaneTurueva**

### CODE REVIEW

File: examples/project/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#################################################################
```
Comment (mosquito) [inline examples/project/Dockerfile]: Dockerfile.ru

File: examples/pypi_project/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#################################################################
```
Comment (mosquito) [inline examples/pypi_project/Dockerfile]: Dockerfile.ru

File: examples/simple/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Берем минимальный по размеру docker контейнер c требуемой версией python
```
Comment (mosquito) [inline examples/simple/Dockerfile]: Dockerfile.ru

**Comment by JaneTurueva** [comment]
@mosquito please check

**Comment by mosquito** [comment]
Great work. Thank you.

**Pull request merged**